recipes-bsp/alsa-state/alsa-state.bbappend

```diff
@@ -0,0 +1,4 @@
+do_install_append_cubox-i() {
```

Comment:
Please add a file with those settings otherwise changes on other layers can break this in the future.